codewithher/CC-Lantern
# 1 feature wrapper

**Why:**

Closes the following issues (all PRs must close an issue):

- closes #1 
- closes #11 
- closes https://github.com/codewithher/CC-Library-Lantern/issues/7
- closes https://github.com/codewithher/CC-Library-Lantern/issues/13

**What's being changed**

Being able to write our own library, we can easily reduce the amount of code that the end user interacts with. This PR address this by implementing the core features that are relevant to our lanterns in a easier to read format.

**Testing instructions**

The easiest way to test is to run activities 1-3 which double as testing files. These run all the wrapper functions (except `wait()`).

You can verify that the code works appropriately by visually inspecting the colors that result from the activity function calls. All the function calls are self-describing with what the end result should look like. Attributes to check for:

- [x] All functions display the appropriate color
- [x] All functions that change brightness visually change brightness
- [x] All functions still work even with invalid time input (underflow and overflow)
- [x] All functions display a color even with invalid color input (underflow and overflow)

**Check off the following**

~~- [ ] I have assigned @totally-not-frito-lays as a reviewer~~
